config/jobs/image-pushing/k8s-staging-public-pii.yaml

```
@@ -0,0 +1,24 @@
+postsubmits:
+  kubernetes/k8s.io:
+    - name: public-log-vendor-matcher-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: wg-k8s-infra-gcb
+        testgrid-tab-name: public-log-vendor-matcher-push-image
+        testgrid-alert-email: public-log-vendor-matcher@ii.coop
+        description: Builds the public-log-vendor-matcher image for generating data that is used for reviewing Kubernetes Public artifact traffic in DataStudio.
+      decorate: true
+      branches:
+        - ^main$
+      spec:
+        serviceAccountName: gcb-builder
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/image-builder:v20210917-ee1e7c845b
+            command:
+              - /run.sh
+            args:
+              # this is the project GCB will run in, which is the same as the GCR images are pushed to.
+              - --project=k8s-infra-public-pii
+              - --scratch-bucket=gs://k8s-staging-public-pii-gcb
```

This should be running in a staging project.  Definitely not in the pii project
```suggestion
              - --project=k8s-staging-infra-tools
              - --scratch-bucket=gs://k8s-staging-infra-tools-gcb
```

---

Updated in 1ffbd4c47d